apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -1035,35 +1035,6 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
                             { resolveMinimalProtocolFormFields() }
                         </Grid.Column>
                     </Grid.Row>
-                    {
```

Comment:
Hope this should be removed from both onprem and cloud, otherwise better to hide the management application through a config